exercises/practice/twelve-days/twelve_days.cpp

```diff
@@ -0,0 +1,7 @@
+#include "twelve_days.h"
+#include <vector>
+#include <string>
+
+namespace twelve_days {
+
```

Comment:
When you run clang-format on all files (you can install and use [pre-commit](https://pre-commit.com) for this, that's what CI uses as well), this will be reformatted to `namespace twelve_days {}`. To avoid this, we add a `TODO` line. Check out some other exercise to see how it is done.